cmd/rosa/main.go

```diff
@@ -17,9 +17,12 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/google/go-github/v41/github"
```

Comment:
Looks like this is pulling quite a big dependency for one http request... could we check the latest version without pulling in the whole GH client into our CLI?

Reply:
@tbrisker  ok i can change it to  https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/rosa/